sql/expression/function/aggregation/common.go

```diff
@@ -0,0 +1,12 @@
+package aggregation
+
+import (
+	"github.com/dolthub/go-mysql-server/sql"
+	"github.com/dolthub/go-mysql-server/sql/expression"
+)
+
+func duplicateExpression(ctx *sql.Context, expr sql.Expression) (sql.Expression, error) {
```

Comment:
Probably should be a public Clone method in the expression package, next to TransformUp